<launch>
<!--<remap from="/camera/rgb/image_raw" to="/xtion/rgb/image_raw"/>
 --><!--<include file="$(find person_detection)/launch/cam.launch"/>
 -->
 
 
  <arg name="launch_prefix" default=""/>
  <arg name="image" default="/xtion/rgb/image_rect_color" />

  <!-- Config and weights folder. -->
  <arg name="yolo_weights_path"          default="$(find darknet_ros)/yolo_network_config/weights"/>
  <arg name="yolo_config_path"           default="$(find darknet_ros)/yolo_network_config/cfg"/>

  <!-- ROS and network parameter files -->
  <arg name="ros_param_file"             default="$(find darknet_ros)/config/ros.yaml"/>
  <arg name="network_param_file"         default="$(find darknet_ros)/config/yolov2-tiny.yaml"/>

  <!-- Load parameters -->
  <rosparam command="load" ns="darknet_ros" file="$(arg ros_param_file)"/>
  <rosparam command="load" ns="darknet_ros" file="$(arg network_param_file)"/>

  <!-- Start darknet and ros wrapper -->
  <node pkg="darknet_ros" type="darknet_ros" name="darknet_ros" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="weights_path"          value="$(arg yolo_weights_path)" />
    <param name="config_path"           value="$(arg yolo_config_path)" />
    <remap from="camera/rgb/image_raw"  to="/xtion/rgb/image_rect_color" />
  </node>


 <!--<include file="$(find darknet_ros)/launch/darknet_ros.launch"/> -->
<node pkg="robocup_human_sensing" type="people_roi.py" name="people_roi"/>
<!-- <node pkg="person_detection" type="extractor.py" name="extractor"/>
 --><!--<node pkg="robocup_human_sensing" type="people_identifier.py" name="people_identifier"/> -->
</launch>

